plugins/woocommerce-admin/client/customer-effort-score-tracks/customer-effort-score-exit-page.ts

```diff
@@ -205,19 +206,63 @@ function getExitPageCESCopy( pageId: string ): {
 					'woocommerce'
 				),
 			};
+		case 'import_products':
+			return {
+				action: pageId,
+				icon: '🔄',
+				noticeLabel: __(
+					'How is your experience with importing products?',
+					'woocommerce'
+				),
+				title: __(
+					`How's your experience with importing products?`,
+					'woocommerce'
+				),
+				description: __(
+					'We noticed you started importing products, then left. How was it? Your feedback will help create a better experience for thousands of merchants like you.',
+					'woocommerce'
+				),
+				firstQuestion: __(
+					'The product import screen is easy to use',
+					'woocommerce'
+				),
+				secondQuestion: __(
+					"The product import screen's functionality meets my needs",
+					'woocommerce'
+				),
+			};
 		default:
 			return null;
 	}
 }
 
+/**
+ * Stores trigger conditions for exit page actions.
+ *
+ * @param {string} pageId page id.
+ */
+function getShouldExitPageFire( pageId: string ) {
+	const conditionPageMap: Record< string, () => boolean > = {
+		import_products: () =>
+			( getQuery() as { page: string } ).page !== 'product_importer',
```

Comment:
This may not be necessary if we handle this in the `product-import-tracking` script, see: https://github.com/woocommerce/woocommerce/pull/35996/files#r1049649526

Reply:
Thanks Lourens, true although I can see an argument for either approach. See my comment https://github.com/woocommerce/woocommerce/pull/35996#discussion_r1050005879.